package.json

```diff
@@ -12,9 +12,10 @@
     "build": "cross-env NODE_ENV=production gulp build",
     "test": "TZ=UTC gulp test",
     "test:unit": "TZ=UTC gulp test:unit",
-    "test:a11y": "gulp test:a11y",
-    "test:integ": "gulp test:integ",
-    "test:motion": "gulp test:motion",
+    "gulp:integ": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" gulp",
```

Comment:
I already added this flag before: https://github.com/cloudscape-design/components/blob/a35b753b47172f9b51123fc3f7c9a627f647112b/build-tools/tasks/unit.js#L19-L23

How about doing the same here, so even when running `npx gulp ...` it still sets the expected options?

Also, I am not sure this line in our contributing docs will remain working: https://github.com/cloudscape-design/components/blob/a35b753b47172f9b51123fc3f7c9a627f647112b/CONTRIBUTING.md?plain=1#L155-L156

Reply:
Nice approach, thanks. Updated the docs too.